tests/core/contracts/conftest.py

```diff
@@ -444,6 +496,21 @@ async def async_math_contract(
     )
 
 
+@pytest_asyncio.fixture(scope="session")
+def async_function_name_tester_contract_abi():
```

Comment:
Since it's a static value with no awaits, we can use the `function_name_tester_contract_abi` fixture for both sync and async tests. No need for an additional async fixture here.

Reply:
Oh right, that makes sense!